patches/tModLoader/Terraria/Wiring.cs.patch

```diff
@@ -50,6 +50,31 @@
  		private static int CurrentUser = 254;
  
  		public static void SetCurrentUser(int plr = -1) {
+@@ -226,21 +_,21 @@
+ 
+ 		public static bool Actuate(int i, int j) {
+ 			Tile tile = Main.tile[i, j];
+-			if (!tile.actuator())
++			if (!TileLoader.CanActuate(tile) || !tile.actuator())
```

Comment:
Isn't this still wrong, wasn't your intent to override vanilla? You'd need different logic:
```suggestion
+			if (!TileLoader.CanActuate(tile) && !tile.actuator())
```